portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,13 @@
-#content {
+/*#content {
```

Comment:
Commenting stuff out isn't usually needed, unless you _really_ think you'll need this later. Even then, you can just pull it back out of source control.

Recommendation: Delete the styles you don't need.